index.html

```diff
@@ -876,41 +876,96 @@ <h3>Directory Service API</h3>
                     incremental transfer of the payloads.
                 </p>
 
+                <p>
+                    The directory APIs include mandatory, recommended, and optional
```

Comment:
How do we relate these to RFC2119 assertions?  How do we capture them in the implementation report if they are not marked up in the text?

Reply:
There are appropriate assertions for those in the text.